src/components/cart/CartPage.js

```diff
@@ -0,0 +1,131 @@
+import React, { useContext, useState } from "react";
+import ListContext from "../../states/ListContext";
+import Space from "../Space";
+import styled from "styled-components";
+import { AstPath } from "prettier";
+import AddToCart from "./AddToCart";
+import Selector from "../productDetail/Selector";
+import CartSelector from "./CartSelector";
+import { Link } from "react-router-dom";
+import ShoppingProd from "./ShoppingProd";
+import CartContext from "../../states/CartContext";
+
+const Container = styled.section`
+  width: 85%;
+  margin: 0 auto;
+`;
+const Tittle = styled.h1``;
+const Wrapper = styled.div``;
+const Top = styled.div``;
+const Total = styled.p`
+  text-align: right;
+`;
+const Medium = styled.div``;
+const ProductsTable = styled.table``;
+const Row = styled.div`
+  display: flex;
+  margin-bottom: 10px;
+  background-color: white;
+`;
+const Image = styled.img`
```

Comment:
All the styled components could live in their own file, maybe `CartPage.styles.jsx`, this improves the readability of the component definition